src/timelines.js

```diff
@@ -843,6 +846,31 @@ export default Kapsule({
         .transition().duration(state.transDuration)
         .call(state.xGrid);
 
+      if (
+        state.showToday
+      ) {
+        const today = new Date();
+        if (
+          today >= state.xScale.domain()[0] &&
+          today <= state.xScale.domain()[1]
+        ) {
+          state.svg.select('line.x-axis-line-today')
+            .style("display", "block")
+            .transition().duration(state.transDuration)
+            .attr("x1", state.xScale(today) + state.leftMargin)
+            .attr("x2", state.xScale(today) + state.leftMargin)
+            .attr("y1", state.topMargin + 1)
+            .attr("y2", state.graphH + state.topMargin)
+            .style("stroke-width", 1)
```

Comment:
These static styles can be added to the CSS instead.

Reply:
Done - good catch.